idaes/power_generation/carbon_capture/mea_solvent_system/properties/tests/test_mea_solvent.py

```diff
@@ -144,7 +144,7 @@ def test_solution(self, model):
         assert pytest.approx(0.0533310121, rel=1e-8) == value(
             model.props[1].surf_tens_phase["Liq"])
 
-        assert pytest.approx(9.83334347e-09, rel=1e-8) == value(
+        assert pytest.approx(8.2258789e-9, rel=1e-8) == value(
```

Comment:
@anujad95 Is this correct? I believe I verified all the model results with the previous model, so I am a bit surprised to see a value change.

Reply:
This is expected because there is a change in the CO2 diffusivity calculation, which I believe, was not implemented in the previous model as well.

Reply:
Do you have a reference for this change and he new values, as well as why it was not implemented in the previous version?

Reply:
The reference for all the property models can be found in the supporting information document of the paper mentioned in the original rigorous MEA column model.

Reply:
Could you please provide a link or send me a copy of the reference and where to look.